pulser-core/pulser/sequence/sequence.py

```diff
@@ -122,7 +122,9 @@ def __init__(
         self._device: BaseDevice = device
         self._in_xy: bool = False
         self._mag_field: Optional[tuple[float, float, float]] = None
-        self._calls: list[_Call] = [_Call("__init__", (register, device), {})]
+        self._calls: list[_Call] = [
+            _Call("__init__", (), {"register": register, "device": device})
```

Comment:
For python they are equivalent because they can be given as `kwargs` or `args`. However, it removes the implicit order of the args, allowing us to invert it in a subclass.